packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -300,6 +300,21 @@ export class SchemaPublisher {
         throw new HiveError(`${project.type} project (${modelVersion}) not supported`);
     }
 
+    const [schemaChecksum, compositeSchemaChecksum, supergraphSchemachecksum] = await Promise.all([
+      this.storage.createSchemaSDLChecksum({
+        targetId: target.id,
+        schemaSDL: sdl,
+      }),
+      this.storage.createSchemaSDLChecksum({
+        targetId: target.id,
+        schemaSDL: checkResult.state?.composition.compositeSchemaSDL,
+      }),
+      this.storage.createSchemaSDLChecksum({
+        targetId: target.id,
+        schemaSDL: checkResult.state?.composition.supergraphSDL,
+      }),
```

Comment:
It feels like this could be a bulk insert operation instead.

```ts
const [
  schemaChecksum,
  compositeSchemaChecksum,
  supergraphSchemachecksum
] = await this.storage.createSchemaSDLChecksums([
  { },
  { },
  { },
])
```